.gitignore

```diff
@@ -1 +1,6 @@
+/vendor
+/phpbench.json
+/composer.lock
+/.php_cs.cache
+/stubs
```

Comment:
`phpbench.json` shouldn't be ignored ..

Reply:
Copied it from somewhere... Sorry about that.